src/components/DeckMechanisms/handleCardEffect.js

```diff
@@ -191,7 +193,11 @@ const handleCardEffect = (state, card, mode) => {
 
     // Archdruid Earyn
     case 'N48': {
-      const spells = state.hand.filter(getPlayableSpells(state))
+      const spells = getPlayableSpells(
```

Comment:
Given all the function does is iterating over the hand and filtering out cards, I would recommend we keep the function as a filtering function.

```js
const spells = state.hand(getPlayableSpells(state))
```